apps/src/templates/studioHomepages/Sections.jsx

```diff
@@ -39,13 +55,26 @@ const Sections = React.createClass({
           isRtl={isRtl}
           description={enrollmentDescription}
         >
+        {this.state.sectionsAction === "join" && this.state.sectionsResult === "success" && (
+          <JoinSectionSuccessNotification sectionName={this.state.sectionsResultName}/>
+        )}
+        {this.state.sectionsAction === "leave" && this.state.sectionsResult === "success" && (
+          <LeaveSectionSuccessNotification sectionName={this.state.sectionsResultName}/>
+        )}
+        {this.state.sectionsAction === "join" && this.state.sectionsResult === "section_notfound" && (
+          <JoinSectionNotFoundNotification sectionId={this.state.sectionsResultName}/>
+        )}
+        {this.state.sectionsAction === "join" && this.state.sectionsResult === "exists" && (
+          <JoinSectionExistsNotification sectionName={this.state.sectionsResultName}/>
+        )}
```

Comment:
I wonder if we should add a fifth notification type for "Unknown error" or "Unexpected error," that we can display if we ever get something truly unexpected back from the server, like a 500.